examples/pytorch/image-classification/create-pytorchjob.ipynb

```diff
@@ -24,8 +26,24 @@
     "The notebook shows how to use Kubeflow Training SDK to create, get, wait, check and delete PyTorchJob."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5e683444",
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "kubeflow_python_sdk=\"git+https://github.com/kubeflow/training-operator.git#subdirectory=sdk/python\"\n",
```

Comment:
For papermill, all the parameters should be in one cell. But this seems to be affecting the readability of the user. How should we improve this?

Reply:
@akshaychitneni @shravan-achar Any thoughts on papermill usage ?
We can pass parameters using `parameters_yaml` only to the single Notebook cell ?

Reply:
@andreyvelich We can pass parameters to multiple Notebook cells. But they should have metadata `parameters` declared and only have parameter definition statements in case of miss injections:)